src/linagora.esn.unifiedinbox/app/services/mailboxes/permissions-service.js

```diff
@@ -81,16 +84,16 @@ angular.module('linagora.esn.unifiedinbox')
       }
       if (!isMyOwnMailbox(mailbox)) {
         return $q.reject(new Error('Only user ' +
-          (mailbox.namespace && mailbox.namespace.owner || '') +
+          (mailbox.namespace && mailbox.namespace.replace(/^.*\[(.*)\]$/g, '$1') || '') +
```

Comment:
copy/paste detected. This magic regex belongs to a dedicated utility method.

Reply:
I put it in a function.